src/DefaultAgent.ts

```diff
@@ -66,4 +65,14 @@ export class DefaultAgent implements IClient {
         options.headers = headers;
         return sendRequest(options, data);
     }
+    async postFormData(
```

Comment:
kdyz je ta funkce uplne stejna jako ta vyse neni lepsi tam pridat jen `string | Buffer | FormData` ?

Reply:
To vyžaduje IClient.